controllers/acsProxyController.go

```diff
@@ -0,0 +1,45 @@
+package controllers
+
+import (
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/UHERO/rest-api/data"
+)
+
+// GetAcsData retrieves 5-Year 2016 Data Profile for all counties and census tracts in the state of Hawaii
+func GetAcsData(cacheRepository *data.CacheRepository) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ids, ok := getAcsIdsList(w, r)
+		if !ok {
+			return
+		}
+		acsKey := os.Getenv("ACS_KEY")
+		joinIds := strings.Join(ids, ",")
+		target := "https://api.census.gov/data/2016/acs/acs5/profile?get=" + joinIds + "&for=tract:*&in=state:15%20county:*&key=" + acsKey
```

Comment:
Let's set this up so that our proxy requires little to no internal knowledge of the ACS api. Then it might make sense for us to use `census` as the namespace instead of `acs`.  so we could have our endpoint look like
```
/v1/census/%s
```
which gets sent over to 
```
api.census.gov/%s
```
without us having to parse the URL.